docs/tutorial_subproc_strings.rst

```diff
@@ -0,0 +1,131 @@
+.. _tutorial_subproc_strings:
+
+************************************
+Tutorial: Subprocess Strings
+************************************
+Strings in xonsh follow two simple rules:
+
+1. Strings in xonsh are always parsed in the same way, and
+2. Python always wins!
+
+Together these rules mean that **even strings in subprocess mode are treated
+like Python strings!** This will (help) preserve your sanity.
+
+No Escape
+=========
+Xonsh strings are exactly like Python strings everywhere. Xonsh uses
+exactly the same escape characters that Python does; no more and no less.
+This is different from other shells, which have a different set of escape
+sequences than Python has.  Notably, many sh-langs allow you to escape
+spaces with ``\ `` (backslash-space).
+
+**bash**
+
+.. code-block:: bash
+
+    $ echo A\ Single\ Argument
+    A Single Argument
+
+In the above example, since the spaces are escaped, the ``echo`` command
+only receves a single argument. Xonsh does not allow this. If you were
+to try this in xonsh, you'd see:
+
+**xonsh**
+
+.. code-block:: xonsh
+
+    $ echo Actually\ Three\ Arguments
+    Actually\ Three\ Arguments
+
+In this example, echo recives three arguments:: ``"Actually\\"``, ``"Three\\"``,
```

Comment:
@scopatz I'm sure this is due to an obvious feature of Python which is not immediately clear to, but perhaps a short note explaining why echo actually receives two backslashes here?

also recives is a misspelling

Reply:
@anki-code I could make a pr if I understood why 🙂